### PR TITLE
ci: add staging site to be tested, liv-4801

### PR DESCRIPTION
**Pull request by jinevo**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,7 +3,10 @@
 
 name: Build and test CI
 
-on: [push]
+on:
+  push:
+  schedule:
+    - cron:  '0 20 17 1/1 *'
```
Comment (tetienne) [inline .github/workflows/ci.yml:9]: If I look the next triggers:

```
next at 2021-08-17 20:00:00
then at 2021-09-17 20:00:00
then at 2021-10-17 20:00:00
then at 2021-11-17 20:00:00
then at 2021-12-17 20:00:00
```

What do you think to run at noon and midnight?

`0 */12 * * *`

Comment (jinevo) [reply .github/workflows/ci.yml:9]: The staging server is stopped during the night. I will change the rules to every day from Monday to Friday at noon: 
`0 12 * * 1-5`

Comment (tetienne) [reply .github/workflows/ci.yml:9]: Perfect.

**Review by Michaelvilleneuve — APPROVED** [review]
:+1:

**Pull request merged**